.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: create release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
+      - name: Build
+        run: |
+          docker-compose build
+          docker-compose up
+      - name: Check iso
+        run: |
+          ls -lrt images/DAppNode-debian-bullseye-amd64.iso
+      - name: Release
+        uses: softprops/action-gh-release@v1
```

Comment:
How is the version of the release decided here?

Reply:
I would say this as something to be discussed.

My opinion:
Hava only one release that will be overwritten on every push to master. Then this repo will contain always the latest ISO of the code in master. The use of this ISO would be for testing purposes, so we do not have to create a new ISO on every single change in the installer code, and we could download it directly from here

Reply:
It's free to create new releases. Overwriting the same release over and over is very strange